src/main.cpp

```diff
@@ -51,8 +51,11 @@ SUA_SERVER    sets and overrides MQTT server address to connect
                 (default is '/data/selfupdates')
 -s, --server    MQTT broker server to connect, has precedence over SUA_SERVER environment variable
                 (default is 'tcp://mosquitto:1883')
--c, --ca        path to certificate to verify connection with bundle server
-                (default is '/etc/ssl/certs/ca-certificates.crt')
+-c, --ca-path   specifies directory containing CA certificates to verify connection with bundle server
+                (default is '/etc/ssl/certs')
```

Comment:
"specifies" is obsolete, removing it would decrease the line length

Reply:
Fixed